R/ncdf.R

```diff
@@ -186,7 +233,32 @@ read_ncdf = function(.x, ..., var = NULL, ncsub = NULL, curvilinear = character(
     d
 }
 
-.get_vars <- function(var, meta) {
+.should_pull <- function(proxy, array_size, num_vars,
+                         n_proxy = options("stars.n_proxy")[[1]] %||% 1.e8) {
```

Comment:
I'm tempted to lower 1e8 for NetCDF. For OPeNDAP data sources this is far far too large. Am tempted to write a heuristic function that sets n_proxy based on sampling the data... thoughts?